sdk/typescript/src/types/version.ts

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+export type RpcApiVersion = {
+  major: number;
+  minor: number;
+  patch: number;
+};
+
+export function parseVersionFromString(
+  version: string
+): RpcApiVersion | undefined {
+  const versions = version.split('.');
+  return {
+    major: parseInt(versions[1]),
```

Comment:
Pass radix in all of these, so `parseInt(versions[1], 10)`

Reply:
Also shouldn't this be indexes `0`, `1`, and `2`?

Reply:
Good catch! Fixed.